web/pdf_viewer.js

```diff
@@ -848,6 +848,7 @@ class PDFViewer {
             console.warn("Warning: XFA-editing is not implemented.");
           } else if (isValidAnnotationEditorMode(mode)) {
             this.#annotationEditorUIManager = new AnnotationEditorUIManager(
+              this.viewer,
               this.container,
```

Comment:
Nit: Please keep the same ordering as elsewhere in the viewer.
```suggestion
              this.container,
              this.viewer,
```